testproject/Assets/Tests/Runtime/MultiprocessRuntime/TestCoordinator.cs

```diff
@@ -86,30 +160,50 @@ public void Update()
             MultiprocessLogger.Log($"quitting application, shouldShutdown set to {m_ShouldShutdown}, is listening {NetworkManager.Singleton.IsListening}, is connected client {NetworkManager.Singleton.IsConnectedClient}");
             if (!m_ShouldShutdown)
             {
-                QuitApplication();
+                QuitApplication($"something wrong happened, was not connected for {Time.time - m_TimeSinceLastConnected} seconds");
                 Assert.Fail($"something wrong happened, was not connected for {Time.time - m_TimeSinceLastConnected} seconds");
             }
         }
+        else if (m_Stopwatch.ElapsedMilliseconds > 5000)
+        {
+            m_Stopwatch.Restart();
+            MultiprocessLogger.Log($"Update - IsInvoking: {NetworkManager.Singleton.IsInvoking()}" +
+                $"IsActiveAndEnabled: {NetworkManager.Singleton.isActiveAndEnabled}" +
+                $" NetworkManager.NetworkConfig.NetworkTransport.name {NetworkManager.NetworkConfig.NetworkTransport.name} " +
+                $" isConnectedClient: {NetworkManager.Singleton.IsConnectedClient};\n" +
+                $" pid: {s_ProcessId}");
+        }
     }
 
-    private static void QuitApplication()
+    private static void QuitApplication(string reason)
     {
 #if UNITY_EDITOR
+        MultiprocessLogger.Log($"Setting UnityEditor isPlaying to false for pid {s_ProcessId} because {reason}");
         UnityEditor.EditorApplication.isPlaying = false;
 #else
+        MultiprocessLogger.Log($"Calling Application.Quit for pid {pid} because: {reason}");
```

Comment:
I think this is causing the compile error:
`MultiprocessLogger.Log($"Calling Application.Quit for pid {pid} because: {reason}");`
Should it be?
`MultiprocessLogger.Log($"Calling Application.Quit for pid {s_ProcessId} because: {reason}");`